org.civicrm.reqangular/src/common/modules/directives.js

```diff
@@ -2,18 +2,21 @@ define([
   'common/angular',
   'common/decorators/ui-select/ui-select-choices',
   'common/decorators/angular-date/datepicker-addon',
+  'common/decorators/ui-bootstrap/uib-tabset',
+  'common/angularBootstrap',
   'common/ui-select',
   'common/modules/templates',
   'common/modules/controllers',
   'common/modules/services',
   'common/modules/apis'
-], function (angular, uiSelectChoicesCustomScrollBars, datepickerAddon) {
+], function (angular, uiSelectChoicesCustomScrollBars, datepickerAddon, uibTabset) {
```

Comment:
There are several points to make here

1. I think it would be better to split this module in two, `common.directives` and `common.decorators`
2. Can you find out what `common/modules/services` and `common/modules/apis` are required for in this case? Why does this module needs to have access to them?
3. Same goes with the angular dependencies that are being injected, is `common.apis` and `ngSanitize` really needed?